crossdock/server/yarpc/server.go

```diff
@@ -65,3 +69,13 @@ func Stop() {
 		fmt.Println("failed to stop:", err.Error())
 	}
 }
+
+func register(rpc transport.Registry) {
```

Comment:
@abhinav I wonder if we should just call this `rpc.Procedures` or `rpc.ProcedureCollection` instead of `transport.Registry` since it's just a map of procedures anyways. I'm a bit wary of just using the term `Registry` by itself because you don't know what it is the registry is storing by looking at it.